src/robotide/editor/kweditor.py

```diff
@@ -568,24 +567,25 @@ def OnKeyDown(self, event):
             if keycode == wx.WXK_SPACE:
                 self._open_cell_editor_with_content_assist()  # Mac CMD
             elif specialkcode in [wx.WXK_DOWN, wx.WXK_UP]:
-                self._skip_except_on_mac(event)
                 self._move_rows(specialkcode)
             elif specialkcode == wx.WXK_RETURN:
-                self._move_cursor_down(event)
-            else:
-                event.Skip()
```

Comment:
Also rewrite this as follows, in order to keep both old and new behavior
```
            elif specialkcode == wx.WXK_RETURN:
                if self.IsCellEditControlShown():
                    event.GetEventObject().WriteText('\n')
                else:
                    self._move_cursor_down(event)
            else:
                event.Skip()
```

Reply:
Recovered old behaviour alt + enter -> move cursor down.

the last event.Skip() is not necessary is because, it eventually will go to the last line "event.Skip()" of method OnKeyDown()

Reply:
Lets leave it like this for now, I'll have to refactor that anyways after your merge as there are some shortcuts that don't need event.Skip().